docs/source/contribute/development/developer_guide/release_instructions.rst

```diff
@@ -0,0 +1,182 @@
+
+7. Once release CI/CD has passed, check the ``skbase`` version on ``pypi``,
+   this should be the new version. It should be checked that all wheels have been
+   uploaded, `here <https://pypi.org/simple/skbase/>`__. As a test, one install
+   of ``skbase`` in a new ``python`` environment should be carried out
+   according to the install instructions (choose an arbitrary version/OS).
+   If the install does not succeed or wheels have not been uploaded, urgent
+   action to diagnose and remedy must be taken. All core developers should be
+   informed of the situation through mail-all in the core developer channel on slack.
+   In the most common case, the install instructions need to be updated.
+   If wheel upload has failed, the tag in 5. needs to be deleted and recreated.
+   The tag can be deleted using the ``git`` command
+   ``git push --delete origin tagname`` from a local repo.
+
+.. _conda_release:
+
+``conda`` release and release validation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+8. If the release on ``pypi`` has succeeded, there should be an automated
+   release PR created against the ``skbase`` conda-forge repo:
+   https://github.com/conda-forge/skbase-feedstock.
+
+  .. note:: Manual creation of release pull request
+     In cases where the release PR is not created automatically it can be created
+     and submitted manually. For general guidelines related to maintaining conda
+     feedstcok packages see
+     `conda-forge package <https://conda-forge.org/docs/maintainer/updating_pkgs.html>`_.
+
+     After forking and cloning the repo, edit the ``meta.yml`` file by:
+
+     - incrementing the version in the line that contains ``{% set version = "0.X.Y" %}``
+     - pasting the sha256 sum of the source archive from github in the
+       ``source/sha256`` section
+
+    Once finished, submit the PR and ask for review.
+
+9. The conda release PR needs to be reviewed and in dependencies should be
+   checked against any changes in the main ``skbase`` repo. In case the dependencies
+   (or python version support) have changes, the ``meta.yml`` file in the conda
+   recipe needs to be updated to reflect those changes.
+
+10. Once reviewed, the conda release PR should merged, and it will automatically
+    trigger a release of the conda package.
+
+11. After 1h, it should be checked whether the package has been released on conda.
+    Once the package is available on ``conda``, a test install should be carried out
+    to validate the release. Should either of these fail, alert the core developers
+    and follow an urgent action plan in line with the description in step 7.
+
+.. _version_number_locs:
+
+Version number locations
+------------------------
+
+Version numbers need to be updated in:
+
+* root ``__init__.py``
+* ``README.md``
+* ``pyproject.toml``
+
+.. _generate_release_notes:
+
+Generating release notes
+------------------------
+
+Release notes can be generated using the ``build_tools.changelog.py`` script,
```

Comment:
is this copy-paste from `sktime`?

If so, it might be problematic, since I think the `changelog` generation tool is broken.
https://github.com/sktime/sktime/issues/3238

I always have to hunt for ca 20% of the PR manually, the exact condition under which a PR gets lost to the tool eludes me at the moment (I suspect it has to do with collection time of the log vs opening time of the PR vs release time, i.e., only a smaller/bigger condition in terms of these three)

Reply:
Yea this was copied from there. Should we remove this then?